generic/tests/trans_hugepage_memory_stress.py

```diff
@@ -42,7 +43,7 @@ def run(test, params, env):
         mem_swap = int(utils_memory.read_from_meminfo('SwapFree')) / 1024
         hugepage_size = (int(utils_memory.read_from_meminfo('Hugepagesize')) /
                          1024)
-        nr_hugetlbfs = (mem_free + mem_swap - mem - qemu_mem) / hugepage_size
+        nr_hugetlbfs = math.ceil((mem_free + mem_swap - mem - qemu_mem) / hugepage_size)
```

Comment:
```suggestion
        nr_hugetlbfs = (mem_free + mem_swap - mem - qemu_mem) // hugepage_size
```

Reply:
Use` // `to take the remainder, but the page needs to be larger than the remainder. So use `ceil` here.

Reply:
`math.ceil()` method rounds a number UP to the nearest integer, but the behavior of `//` rounds a number DOWN, which aligns to `/` in python2.
So this is a change in history, any background? @huangyum , what's your opinion?

Reply:
We should set the hugepage number at least as required. So I prefer to use math.ceil. Thanks.

Reply:
OK, thanks all.